darkdetect/_linux_detect.py

```diff
@@ -0,0 +1,37 @@
+#-----------------------------------------------------------------------------
+#  Copyright (C) 2019 Alberto Sottile
+#
+#  Distributed under the terms of the 3-clause BSD License.
+#-----------------------------------------------------------------------------
+
+import subprocess
+
+
+def theme():
+    # Here we just triage to GTK settings for now
+    try:
+        proc = subprocess.run(
+            ['gsettings', 'get', 'org.gnome.desktop.interface',
+             'gtk-theme'], capture_output=True)
+        theme = proc.stdout.decode().strip().strip("'")
+    except Exception:
+        return None
+    else:
+        if theme.endswith('-dark'):
+            return 'Dark'
+        else:
+            return 'Light'
+
+def isDark():
+    got = theme()
+    if got is None:
+        return None
```

Comment:
On macOS and Windows, the package idea is to always return `'Light'` whenever the detection fails on a supported platform, to provide a nice fallback to dependent code. See e.g. https://github.com/albertosottile/darkdetect/blob/d4a5d2c3ce5611834818cbd229d7095b248f9f9a/darkdetect/_mac_detect.py#L55-L58 
`None` should be returned only by `_dummy` and so, only from unsupported platforms. 

Hence, I would adapt this and the next two functions according to this behavior.

Reply:
Pushed a commit to address this and the other comments, let me know what you think!

Reply:
Looks great, thank you 👍